pkgs/development/compilers/llvm/7/llvm.nix

```diff
@@ -10,15 +10,17 @@
 , version
 , release_version
 , zlib
+, buildPackages
 , debugVersion ? false
 , enableManpages ? false
 , enableSharedLibraries ? true
-, targets ? [ stdenv.hostPlatform stdenv.targetPlatform ]
-, enableWasm ? true # TODO fold this into `targets` somehow
```

Comment:
I did not want to have this as an API, because it should be in `targets` once it leaves the experimental stage.